Issue when used in webpack build with redux-actions > v0.13.0, roux-webpack-resolver-plugin

Redux actions > v0.11.0 requires lodash methods with `require('lodash/toString')`. This causes the member-profile build to fail with the following error:

```
ERROR in ./~/redux-actions/lib/combineActions.js
Module not found: TypeError: Cannot read property 'javaScript' of undefined
 @ ./~/redux-actions/lib/combineActions.js 21:16-42
```

For some reason, @retailmenot/roux is erroneously determining that lodash is a pantry, and trying to treat it as such. This should not be happening.

Additional output when running with `DEBUG=*`

```
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "identity"] message: 'Pantry "lodash" has no ingredient "identity"' } +21ms
  roux:sdk undefined +1ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "isFunction"] message: 'Pantry "lodash" has no ingredient "isFunction"' } +0ms
  roux:sdk undefined +0ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "identity"] message: 'Pantry "lodash" has no ingredient "identity"' } +0ms
  roux:sdk undefined +0ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "isNil"] message: 'Pantry "lodash" has no ingredient "isNil"' } +0ms
  roux:sdk undefined +0ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "includes"] message: 'Pantry "lodash" has no ingredient "includes"' } +0ms
  roux:sdk undefined +0ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "isString"] message: 'Pantry "lodash" has no ingredient "isString"' } +0ms
  roux:sdk undefined +0ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "isFunction"] message: 'Pantry "lodash" has no ingredient "isFunction"' } +0ms
  roux:sdk undefined +0ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "isEmpty"] message: 'Pantry "lodash" has no ingredient "isEmpty"' } +1ms
  roux:sdk undefined +1ms
  roux:sdk got error [TypeError: Cannot read property 'javaScript' of undefined] +2ms
  roux:sdk TypeError: Cannot read property 'javaScript' of undefined
  roux:sdk     at /Users/lzilioli/Projects/member-profile/node_modules/@retailmenot/roux/index.js:413:38
  roux:sdk     at tryCatcher (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/util.js:16:23)
  roux:sdk     at Promise._settlePromiseFromHandler (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/promise.js:510:31)
  roux:sdk     at Promise._settlePromise (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/promise.js:567:18)
  roux:sdk     at Promise._settlePromise0 (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/promise.js:612:10)
  roux:sdk     at Promise._settlePromises (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/promise.js:691:18)
  roux:sdk     at Promise._fulfill (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/promise.js:636:18)
  roux:sdk     at SomePromiseArray.PromiseArray._resolve (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/promise_array.js:125:19)
  roux:sdk     at SomePromiseArray._promiseFulfilled (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/some.js:58:18)
  roux:sdk     at Promise._settlePromise (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/promise.js:572:26)
  roux:sdk     at Promise._settlePromise0 (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/promise.js:612:10)
  roux:sdk     at Promise._settlePromises (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/promise.js:691:18)
  roux:sdk     at Async._drainQueue (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/async.js:133:16)
  roux:sdk     at Async._drainQueues (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/async.js:143:10)
  roux:sdk     at Immediate.Async.drainQueues [as _onImmediate] (/Users/lzilioli/Projects/member-profile/node_modules/bluebird/js/release/async.js:17:14)
  roux:sdk     at processImmediate [as _immediateCallback] (timers.js:383:17) +1ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "isSymbol"] message: 'Pantry "lodash" has no ingredient "isSymbol"' } +0ms
  roux:sdk undefined +1ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "camelCase"] message: 'Pantry "lodash" has no ingredient "camelCase"' } +0ms
  roux:sdk undefined +0ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "identity"] message: 'Pantry "lodash" has no ingredient "identity"' } +0ms
  roux:sdk undefined +0ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "isPlainObject"]
  message: 'Pantry "lodash" has no ingredient "isPlainObject"' } +0ms
  roux:sdk undefined +0ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "isArray"] message: 'Pantry "lodash" has no ingredient "isArray"' } +0ms
  roux:sdk undefined +0ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "reduce"] message: 'Pantry "lodash" has no ingredient "reduce"' } +0ms
  roux:sdk undefined +0ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "isString"] message: 'Pantry "lodash" has no ingredient "isString"' } +0ms
  roux:sdk undefined +0ms
  roux:sdk got error { [Error: Pantry "lodash" has no ingredient "isFunction"] message: 'Pantry "lodash" has no ingredient "isFunction"' } +0ms
  roux:sdk undefined +0ms
```

--- a/index.js
+++ b/index.js
@@ -399,11 +399,11 @@ function resolve(pantry, ingredient, entryPoint, config) {
 			return pantryInstance;
 		}
 
-		var ingredientInstance = pantryInstance.ingredients[ingredient];
-
-		if (!ingredientInstance) {
+		if (!pantryInstance.ingredients.hasOwnProperty(ingredient)) {
 			throw new IngredientDoesNotExistError(pantry, ingredient);
 		}
+
+		var ingredientInstance = pantryInstance.ingredients[ingredient];
 
 		if (!entryPoint) {
 			// we are resolving an ingredient, so we're done

